app/parse-siti-payment-file/schemas/payment-request.js

```diff
@@ -7,10 +10,10 @@ module.exports = Joi.object({
   frn: Joi.number().integer().min(1000000000).max(9999999999).required(),
   marketingYear: Joi.number().integer().greater(2015).less(2099).required(),
   paymentRequestNumber: Joi.number().integer().required(),
-  agreementNumber: Joi.string().required(),
+  agreementNumber: Joi.string().optional(),
```

Comment:
Lump Sums does not have an agreement number in payment file.  `enrichment` service will map `contractNumber` to `agreementNumber` for this scheme so no action needed downstream.

All we need to do in this repo is make it optional.